pixels-common/src/main/resources/pixels.properties

```diff
@@ -58,6 +58,7 @@ cache.read.direct=false
 # properties for pixels writer
 pixel.stride=10000
 row.group.size=268435456
+row.group.align=32
```

Comment:
Rename it to column.chunk.alignment, and add a line of comment above it to say it is for SIMD and its unit is byte.